samples/RazorComponentSample/App.razor

```diff
@@ -1,5 +0,0 @@
-﻿@*
```

Comment:
Are these files removed intentionally?

Reply:
Yes. It's some middle version of Blazor sample and somehow not removed clean...